src/pkgcheck/checks/profiles.py

```diff
@@ -28,6 +28,20 @@ def desc(self):
         return f'{self.path!r}: unknown package: {self.atom!r}'
 
 
+class UnmatchedProfilePackageUnmask(results.ProfilesResult, results.Warning):
+    """Profile files includes package entry unmask that isn't masked in the
```

Comment:
"The profile's files include a package.unmask (or similar) entry which negates a non-existent mask, i.e. it undoes a mask which doesn't exist in the parent profile."